dbos/dbos-config.schema.json

```diff
@@ -24,43 +24,51 @@
         "additionalProperties": false,
         "properties": {
           "hostname": {
-            "type": "string",
-            "description": "The hostname or IP address of the application database"
+            "type": ["string", "null"],
```

Comment:
Should we just deprecate the entire `database` field?

Reply:
We need to keep it for the migrations steps -- I think we could deprecate the system db name tho. Let's chat